public/js/dom.js

```diff
@@ -1,44 +1,56 @@
 // const url = `https://api.themoviedb.org/3/search/movie?api_key=${config.apiKey }&language=en-US&query=`;
-function selecElement(id) {
-	return document.querySelector(id);
-}
-function addEventListenerForSelector(selectorElement, event, callback){
-	return document.querySelector(selectorElement).addEventListener(event,callback);
-}
-function createElement(e){
-	return document.createElement(e);
-}
-addEventListenerForSelector('#submit-btn', 'submit', (event) => {
-	event.preventDefault();
-	if (selecElement('#searchInput').value == ""){
-		alert('Entar a movie name');
+const submitBtn = document.querySelector('#submit-btn');
+const searchInput = document.querySelector('#searchInput');
+const container = document.querySelector('#movies');
+
+// Create Node function
+function makeNodes(nodeName, nodeType) {
+	if (nodeName.length !== nodeType.length) {
+		return 'Error';
+	} else {
+		const node = { };
+		nodeName.map((name, index) => {
```

Comment:
always make you function shorter as can as possible 
and here there are some errors 
Like :
```
const nodes = { } ;
nodeName.map((name, index) => 
nodes[nodeName[index]] = document.createElement(nodeType[index]);

```